src/sentry/issues/attributes.py

```diff
@@ -200,25 +212,34 @@ def post_save_log_group_attributes_changed(instance, sender, created, *args, **k
             _log_group_attributes_changed(Operation.CREATED, "group", None)
             send_snapshot_values(None, instance, False)
         else:
-            update_fields = kwargs.get("update_fields", set())
-            if not update_fields:
-                # we have no guarantees update_fields is used everywhere save() is called
-                # we'll need to assume any of the attributes are updated in that case
-                attributes_updated = {"all"}
-            else:
-                attributes_updated = {"status", "substatus", "num_comments"}.intersection(
-                    update_fields or ()
-                )
-            if attributes_updated:
-                _log_group_attributes_changed(
-                    Operation.UPDATED, "group", "-".join(sorted(attributes_updated))
-                )
+            if process_update_fields(kwargs.get("update_fields", set())):
                 send_snapshot_values(None, instance, False)
-
     except Exception:
         logger.exception("failed to log group attributes after group post_save")
 
 
+@receiver(post_update, sender=Group, dispatch_uid="post_update_group", weak=False)
+def post_update_group(sender, updated_fields, model_ids, *args, **kwargs):
+    try:
+        updated_fields = process_update_fields(updated_fields)
+        if updated_fields:
+            bulk_send_snapshot_values(model_ids, None)
+    except Exception:
+        logger.exception("failed to log group attributes after group_owner updated")
+
+
+def process_update_fields(updated_fields):
+    if not updated_fields:
+        # we have no guarantees update_fields is used everywhere save() is called
+        # we'll need to assume any of the attributes are updated in that case
+        updated_fields = {"all"}
+    else:
+        updated_fields = {"status", "substatus", "num_comments"}.intersection(updated_fields or ())
```

Comment:
nit: valid fields as a separate thing for readability 
```suggestion
        VALID_FIELDS = {"status", "substatus", "num_comments"}
        updated_fields = VALID_FIELDS.intersection(updated_fields or ())
```